### PR TITLE
bpo-41776: Revise example of "continue" in the tutorial documentation

### PR DESCRIPTION
**Pull request by neerajsamtani**
https://bugs.python.org/issue41776

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).



## CLA Missing

Our records indicate the following people have not signed the CLA:

@neerajsamtani

For legal reasons we need all the people listed to sign the CLA before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.

If you have recently signed the CLA, please wait at least one business day
before our records are updated.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

File: Doc/tutorial/controlflow.rst [hunk 1]
```diff
@@ -210,7 +210,7 @@ iteration of the loop::
     ...     if num % 2 == 0:
     ...         print("Found an even number", num)
     ...         continue
-    ...     print("Found a number", num)
+    ...     print("Found an odd number", num)
     Found an even number 2
     Found a number 3
```
Comment (ericvsmith) [inline Doc/tutorial/controlflow.rst]: The output needs to be changed, too: "Found an odd number 3", etc.

**Review by ericvsmith — CHANGES_REQUESTED** [review]

**Comment by neerajsamtani** [comment]
I have made the requested changes; please review again

**Review by ericvsmith — COMMENTED** [review]
This looks good. As soon as the CLA is signed, I'll merge it.

**Comment by neerajsamtani** [comment]
@ericvsmith notifying you that the CLA has been signed so you can now merge it. Thanks

**Comment by ericvsmith** [comment]
Thanks!

**Pull request merged**